src/components/LenderProfile/LenderLoansList.vue

```diff
@@ -1,18 +1,25 @@
 <template>
 	<section v-if="lenderLoans.length > 0" class="tw-my-8" id="lender-loans">
-		<h4 class="data-hj-suppress tw-mb-1">
-			{{ lenderLoansTitle }}
-		</h4>
-		<p class="tw-mb-2">
-			{{ showedLoans }}
-		</p>
+		<div v-if="totalCount > 0">
```

Comment:
Should this be a loading flag instead of `totalCount`? Since what if the lender has no loans, the loaders will never disappear.

Reply:
From line 2, we are not showing the whole section if the lender has no loans.